### DISCUSSION TITLE
Proxmox troubleshot

### DISCUSSION THREAD

**Discussion by main22** [community/community#67240] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i have trouble on proxmox please help me for create SSH with cloudflare

**Comment by pnjrmdn** [comment]
How to Make Localhost Accessible Online | Accessible from the Internet Network https://lnkd.in/gkAPEWnv

**Comment by pnjrmdn** [comment]
Setup Cloudflare Zero Trust Network and connect to an SSH server
https://lnkd.in/gf4dtfPR

**Comment by main22** [comment]
nice

**Reply by pnjrmdn** [reply]
ur welcome

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
